webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -836,36 +866,34 @@ def _update_graph(
                     for trace in add_observation_trace(self.observations.get(vector)):
                         fig.add_trace(trace, i + 1, 1)
 
-            fig = fig.to_dict()
-            fig["layout"].update(
-                height=800,
-                margin={"t": 20, "b": 0},
-                barmode="overlay",
-                bargap=0.01,
-                bargroupgap=0.2,
-            )
-            fig["layout"] = self.theme.create_themed_layout(fig["layout"])
-
-            if "Histogram" in trace_options:
-                # Remove linked x-axis for histograms
-                if "xaxis2" in fig["layout"]:
-                    fig["layout"]["xaxis2"]["matches"] = None
-                    fig["layout"]["xaxis2"]["showticklabels"] = True
-                if "xaxis4" in fig["layout"]:
-                    fig["layout"]["xaxis4"]["matches"] = None
-                    fig["layout"]["xaxis4"]["showticklabels"] = True
-                if "xaxis6" in fig["layout"]:
-                    fig["layout"]["xaxis6"]["matches"] = None
-                    fig["layout"]["xaxis6"]["showticklabels"] = True
+                fig = fig.to_dict()
+                fig["layout"].update(
+                    height=800,
+                    margin={"t": 20, "b": 0},
+                    barmode="overlay",
+                    bargap=0.01,
+                    bargroupgap=0.2,
+                )
+                fig["layout"] = self.theme.create_themed_layout(fig["layout"])
+
+                if "Histogram" in trace_options:
+                    # Remove linked x-axis for histograms
+                    if "xaxis2" in fig["layout"]:
+                        fig["layout"]["xaxis2"]["matches"] = None
+                        fig["layout"]["xaxis2"]["showticklabels"] = True
+                    if "xaxis4" in fig["layout"]:
+                        fig["layout"]["xaxis4"]["matches"] = None
+                        fig["layout"]["xaxis4"]["showticklabels"] = True
+                    if "xaxis6" in fig["layout"]:
+                        fig["layout"]["xaxis6"]["matches"] = None
+                        fig["layout"]["xaxis6"]["showticklabels"] = True
```

Comment:
this extra indent looks funky. Sure this is correct?

Reply:
Hehe, I'll better gonna consult my local pylint installation ;)

Reply:
Good catch!